templates/vue/src/components/Lightbox/media/TapestryMedia.vue

```diff
@@ -123,10 +124,12 @@ export default {
     },
   },
   beforeDestroy() {
-    this.updateNodeProgress({
-      id: this.nodeId,
-      progress: this.node && this.node.progress,
-    })
+    if (!Helpers.nodeAndUserAreDyad(this.node)) {
```

Comment:
It's better to centralize this check in the `updateNodeProgress` action itself so we don't have to do the check everywhere. You may need to change that function to accept the whole node rather than its ID so you can pass that to the helper function - otherwise you will need to get the node manually from the store and pass it (whatever you think is cleaner).

Reply:
Yep, moved the checks into updateNodeProgress